runtime/caching/cache.go

```diff
@@ -90,6 +97,8 @@ func (c *Cache) Require(ctx Context, options interface{}) (*Handle, error) {
 
 		// Take the entry
 		debug("cache entry '%s' found in cache, refCount: %d", optionsHash, e.refCount+1)
+		c.countCachehits++
+		c.cachehit.Count("count cache-hit", c.countCachehits)
```

Comment:
Use the `monitor` given in `New` and then call `monitor.Count('cache-hit', 1)`

If there is a cache miss, the call `monitor.Count('cache-miss', 1)`

the monitor will aggregate metrics that have the same name. Take a look at statsum if you're curious.